save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/controllers/TestSuitesPreprocessorController.kt

```diff
@@ -18,6 +19,9 @@ import org.springframework.web.bind.annotation.RestController
 import reactor.core.publisher.Mono
 import kotlin.io.path.div
 
+typealias TestSuiteList = List<TestSuite>
+typealias CloneAndProcessDirectory = GitPreprocessorService.(GitDto, String, GitRepositoryProcessor<TestSuiteList>) -> Mono<TestSuiteList>
```

Comment:
```suggestion
typealias CloneAndProcessDirectoryAction = GitPreprocessorService.(GitDto, String, GitRepositoryProcessor<TestSuiteList>) -> Mono<TestSuiteList>
```
Maybe something like this?

Reply:
agree. changed